WhaleSpotting/react-app/src/styles/ReportSighting.scss

```diff
@@ -0,0 +1,90 @@
+@import "./Constants.scss";
+.report-sighting{
+    font-family: $fontFamily;
+
+    .banner-image{
+        height: 40vh;
+        object-fit: cover;
+        width: 100%;
+    }
+
+    .required{
+        color: $secondaryDark;
+        font-size: $mobileBodyFontSize;
+    }
+
+    .container{
+        margin: auto;
+        max-width: 800px;
+        padding: 4% 5%;
+    }
+
+    .card-component{
+        background: $primaryCard;
+        border-radius: 5px;
```

Comment:
Don't think we need any of this, the other card styling is enough

Reply:
removed